Source/WebCore/workers/service/WorkerSWClientConnection.h

```diff
@@ -26,12 +26,19 @@
 #pragma once
 
 #include "SWClientConnection.h"
+#include <wtf/Forward.h>
+#include <wtf/ObjectIdentifier.h>
 
 namespace WebCore {
 
 class WorkerGlobalScope;
 class WorkerThread;
 
+struct CookieChangeSubscription;
+
+struct SWClientRequestIdentifierType;
+using SWClientRequestIdentifier = AtomicObjectIdentifier<SWClientRequestIdentifierType>;
```

Comment:
Probably this should be private, inside the class scope. I don't see any reason to expose this globally or even outside the class?